Cargo.lock

```diff
@@ -5025,7 +5025,7 @@ dependencies = [
 
```

Comment:
> This is blocked on #6624

The missed dependency update is also blocked on PR #6624:

> 1. Run cargo update on the latest main branch, and keep the output
> 2. If needed, update [deny.toml](https://github.com/ZcashFoundation/zebra/blob/main/book/src/dev/continuous-integration.md#fixing-duplicate-dependencies-in-check-denytoml-bans)
> 3. Open a separate PR with the changes, including the output of cargo update

Did you want some help doing this? I think it will be easier in a separate PR.

Reply:
Ok, yea. sorry about that an thanks.

Reply:
This has been merged into this PR, the exact changes are in PR #6656